.github/workflows/gitlab.yml

```diff
@@ -31,4 +31,4 @@ jobs:
           project-id: ${{ secrets.PROJECT_ID }}
           schedule: ${{ github.event_name == 'schedule' }}
           cancel-outdated-pipelines: ${{ github.ref_name != 'main' }}
-          triggered-ref: v0.3.1
+          triggered-ref: LLT-3951_ipv6_analytics
```

Comment:
Adding a message not to forget to change this to actual tag once that is merged in `libtelio-build`